pkg/userdata/ubuntu/testdata/vsphere-proxy.yaml

```diff
@@ -0,0 +1,360 @@
+#cloud-config
+
+hostname: node1
+
+
+ssh_pwauth: no
+ssh_authorized_keys:
+- "ssh-rsa AAABBB"
+
+write_files:
+- path: "/etc/environment"
+  content: |
+    PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games"
+    HTTP_PROXY=http://192.168.100.100:3128
+    http_proxy=http://192.168.100.100:3128
```

Comment:
Nit: Can be simplified as `http_proxy=$HTTP_PROXY`

Reply:
Variable expansion does not work in /etc/environment